korge-core/build.gradle.kts

```diff
@@ -5,6 +5,8 @@ dependencies {
         //add("androidMainApi", "androidx.javascriptengine:javascriptengine:1.0.0-alpha05")
         //add("androidMainApi", "com.google.guava:guava:31.0.1-android")
     }
+    commonMainApi(project(":korlibs-datastructure"))
+    commonMainApi(project(":korlibs-platform"))
```

Comment:
This should be removed. I mean, the idea is that those artifacts are transitively resolved via `korge-foundation` that is already included.